src/components/interface/cbuf_mgr/cbuf_mgr.h

```diff
@@ -68,9 +77,28 @@ struct cbuf_shared_page {
 	struct cbuf_ring_element buffer[CSP_BUFFER_SIZE];
 };
 
-/* GAP #include <cbuf_vect.h> */
-/* #include <mem_mgr_large.h> */
-/* /\* Included mainly for struct cbuf_meta: *\/ */
-/* #include "../cbuf_c/tmem_conf.h" */
+#define OP_NUM 10
+typedef enum {
+	CBUF_CRT = 0,
+	CBUF_COLLECT,
+	CBUF_DEL,
+	CBUF_RETRV,
+	CBUF_REG,
+	CBUF_MAP
+} cbuf_debug_t;
```

Comment:
`OP_NUM` should be named consistently to the rest here.  It should also be the last item in the enum so that if you add operations, it is automatically updated.  e.g.

```
typedef enum {
      CBUF_OP_CRT, 
      CBUF_OP_MAP,
      CBUF_OP_MAX
} cbuf_ops_t;
```

Please note how everything is consistent, and max is automatically correct (it replaces `OP_NUM`).